src/libsync/creds/credentialmanager.cpp

```diff
@@ -138,7 +138,7 @@ QStringList CredentialManager::knownKeys(const QString &group) const
 
 QSettings *CredentialManager::credentialsList() const
 {
-    // delayed init as scope requires a fully inizialised acc
+    // delayed init as scope requires a fully initialised acc
```

Comment:
I personally speak en-US, but for this repository, it doesn't look like I check for en-GB/en-US. And in general, I try not to meddle w/ locale preferences. That said, he `z` at the beginning gave me momentary pause.